tools/razzle.cmd

```diff
@@ -3,6 +3,7 @@
 rem Open Console build environment setup
 rem Adds msbuild to your path, and adds the open\tools directory as well
 rem This recreates what it's like to be an actual windows developer!
+rem Copyright 2019 Microsoft

```

Comment:
you are sure for copyright with the year? because each year it will be necessary to make a sweater request a little useless to increase this value.....  :/